### PR TITLE
README: add the appArmor related document [CI SKIP]

### PR DESCRIPTION
**Pull request by Vicente-Cheng**
**IMPORTANT: Please do not create a Pull Request without creating an issue first.**

**Problem:**
`make ci` failed in some local environment

**Solution:**
Give a clear hint for how to run the `make ci` locally

**Related Issue:**
https://github.com/harvester/harvester/issues/6070

**Test plan:**
No need

### CODE REVIEW

File: tests/README.md [hunk 1]
```diff
@@ -5,6 +5,22 @@ This path contains the basic testing framework and integration testing cases, th
 ### Integration test
 The integration test is performed on every pull request and is run by the Drone CI. The script is stored in the `scripts/test-integration` file.
 
+Before the integration test, you need to check the apparmor status to ensure the `usr.sbin.virtqemud` is not enabled.
+Or you might see the following error when you test VM API
```
Comment (bk201) [inline tests/README.md]:
```suggestion
Before the integration test, you must check the AppArmor status to ensure the `usr.sbin.virtqemud` is not enabled.
Or you might see the following error in virt-launcher pods when you test the VM API

```
A tiny suggestion to clarify where the log is.

**Review by bk201 — APPROVED** [review]

File: tests/README.md [hunk 2]
```diff
@@ -5,6 +5,22 @@ This path contains the basic testing framework and integration testing cases, th
...
+```
+
+You can use the following command to check:
+```
+$ aa-status |grep virtqemud
+```
+
+And use the following command to disable:
+```
+$ aa-disable /etc/apparmor.d/usr.sbin.virtqemud
```
Comment (brandboat) [inline tests/README.md]: So we don't need to disable libvirtd as well ?

BTW I need to use sudo to make `aa-disable /etc/apparmor.d/usr.sbin.virtqemud` work.

**Review by brandboat — APPROVED** [review]
LGTM, thank you 😃

File: tests/README.md (hunk 2 above)
Comment (Vicente-Cheng) [reply tests/README.md]: yeah, I tested, we only need to disable `virtqemud`.

I will add the sudo... thanks for the reminder!

Comment (brandboat) [reply tests/README.md]: > yeah, I tested, we only need to disable virtqemud.

I see. Thanks for clarifying. 👍

**Comment by Vicente-Cheng** [comment]
This is only a document change. Seems the [CI SKIP] is not working. I go ahead merge it because all reviewers approved.

**Comment by brandboat** [comment]
> Seems the [CI SKIP] is not working

Yeah... it works only when adding `ci skip` in commit message
quoted from https://docs.github.com/en/actions/managing-workflow-runs/skipping-workflow-runs
> Workflows that would otherwise be triggered using on: push or on: pull_request won't be triggered if you add any of the following strings to the commit message in a push, or the HEAD commit of a pull request:

**Pull request merged**